game_app/app.py

```diff
@@ -10,10 +11,13 @@ def __init__(self):
         self._games: Dict[str, TicTacToeGame] = {}
 
     def start_game(self, first_player_id: str, second_player_id: str) -> TicTacToeGameInfo:
-        raise NotImplementedError
+        game_id = str(uuid4())
+        self._games[game_id] = TicTacToeGame(\
```

Comment:
Кажется, форматирование здесь не так должно выглядеть.
```suggestion
        self._games[game_id] = TicTacToeGame(
```